### PR TITLE
Fix Roster Auth

### PR DESCRIPTION
**Pull request by tarebyte**
We didn't add the `OrganizationAuthorization` module to the `RostersController`. So in theory a user could modify another classrooms Roster if they had the proper URL.

### CODE REVIEW

File: app/controllers/rosters_controller.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 class RostersController < ApplicationController
-  before_action :ensure_student_identifier_flipper_is_enabled, :set_organization
+  include OrganizationAuthorization
```
Comment (ptoomey3) [inline app/controllers/rosters_controller.rb]: Just to double check...are there any other controllers that should include this but don't currently?

Comment (tarebyte) [reply app/controllers/rosters_controller.rb]: @ptoomey3 nope that was the only one I found, I went through the rest of them when I did this PR.

Comment (ptoomey3) [reply app/controllers/rosters_controller.rb]: 👍 - I figured as much, but just had to ask the question :smile:.

**Pull request merged**